src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.SemanticChangeProcessor.cs

```diff
@@ -48,6 +48,20 @@ private sealed class SemanticChangeProcessor : IdleProcessor
                     _pendingWork = new Dictionary<DocumentId, Data>();
 
                     Start();
+
+                    AsyncProcessorTask.ContinueWith(
+                        _ =>
+                        {
+                            foreach (var (documentId, data) in _pendingWork)
```

Comment:
>documentId, [](start = 42, length = 11)

Consider using `_` instead of `documentId` to make it clear it's not used.

Reply:
:arrow_right: Fixed in 9c28f4a